src/site/content/en/blog/performance-measure-memory/index.md

```diff
@@ -0,0 +1,357 @@
+---
+title: Monitor your web page's total memory usage with `performance.measureMemory()`
+subhead: >
+  Learn how to measure memory usage of your web page in production to detect regressions.
+description: >
+  Learn how to measure memory usage of your web page in production to detect regressions.
+date: 2020-04-07
+authors:
+  - ulan
+hero: hero.jpg
+alt: >
+  Green RAM stick. Photo by Harrison Broadbent on Unsplash.
+origin_trial:
+    url: https://developers.chrome.com/origintrials/#/view_trial/1281274093986906113
+tags:
+  - post
+  - memory
+  - javascript
+  - capabilities
```

Comment:
```suggestion
```

This API doesn't seem to be part of the Capabilities project: https://docs.google.com/spreadsheets/d/1de0ZYDOcafNXXwMcg4EZhT0346QM-QFvZfoD8ZffHeA/edit#gid=557099940